src/Components/Footer.js

```diff
@@ -0,0 +1,9 @@
+const Footer = () => {
+    return (
+        <footer>
+            <p>Created by <a href="https://itsoliviasparks.com/" target="_blank" rel="noreferrer">Olivia Sparks</a> at <a href="https://junocollege.com" target="_blank" rel="noreferrer">Juno College</a>. Data provided by <a href="https://www.nps.gov/subjects/developer/api-documentation.htm" target="_blank" rel="noreferrer">The U.S. National Park Service</a>.</p>
```

Comment:
Accessibility note: Your footer is also < 16px so it looks pretty small on my screen. I wanna see who made this cool app!! 😎